js-inputs-and-string/js-inputs-and-strings_change-box-appearance/js/index.js

```diff
@@ -16,3 +16,25 @@ For further information check MDN:
 - about CSS declarations with "style": https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
 
 */
+const box = document.querySelector('[data-js="box"]');
+const colorSlider = document.querySelector('[data-js="input-color"]');
+const radiusSlider = document.querySelector('[data-js="input-radius"]');
+const rotationSlider = document.querySelector('[data-js="input-rotation"]');
+
+
+colorSlider.addEventListener("change", (event) => {
```

Comment:
if you pick the event listenet "input" it while already change while you slide

Reply:
what is this mean exactly? can you show me example?

Reply:
it means that for example the color while change WHILE you slide. It's actually easier to try it out and see for yourself than explaining it.